client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -143,7 +151,7 @@
                 $scope.origDictionary = result;
                 $scope.dictionary = _.create(result);
                 $scope.dictionary.content = _.create(result.content || {});
-                $scope.dictionary.is_active = $scope.dictionary.is_active !== false;
+                $scope.dictionary.is_active = $scope.dictionary.is_active !== 'false';
```

Comment:
This was the issue, on upload file the boolean is interpreted as string on server and not as a boolean.
So changed the data to be string instead of boolean.